packages/network/package.json

```diff
@@ -4,24 +4,23 @@
   "version": "1.41.0",
   "source": "src/index.ts",
   "main": "src/index.ts",
+  "type": "module",
   "engines": {
-    "node": ">=16"
+    "node": ">=18"
```

Comment:
ooc what is this used for? do we need it in the package's `package.json` in addition to the root `package.json` and if so do we also need the `pnpm` record here?

Reply:
I tried to remove this, but then parcel failed to build due to some issue with `threads`. Adding this back in magically solved it and unclear why.
https://github.com/latticexyz/mud/pull/591/commits/98305000dbac100d5e68a8d4afb785034165f238
This is also why I added https://github.com/latticexyz/mud/issues/612